eng/Version.Details.xml

```diff
@@ -1,7 +1,44 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
-  <ProductDependencies></ProductDependencies>
+  <ProductDependencies>
+    <Dependency Name="Microsoft.Extensions.CommandLineUtils.Sources" Version="3.0.0-preview.19069.2">
+      <Uri>https://github.com/aspnet/Extensions</Uri>
+      <Sha>36471a9622b74a43185915550907aa28605be099</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.HashCodeCombiner.Sources" Version="3.0.0-preview.19069.2">
+      <Uri>https://github.com/aspnet/Extensions</Uri>
+      <Sha>36471a9622b74a43185915550907aa28605be099</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.NonCapturingTimer.Sources" Version="3.0.0-preview.19069.2">
+      <Uri>https://github.com/aspnet/Extensions</Uri>
+      <Sha>0000000000000000000000000000000000000000</Sha>
```

Comment:
Did you run update-dependencies on this?  Why is this an invalid sha?

Reply:
No I put this in manually.

Reply:
@natemcmaster @rynowak If that includes extensions then you can leave as 000 for now.